ephios/user_management/models.py

```diff
@@ -131,6 +134,30 @@ def get_shifts(self, with_participation_state_in):
         ).values_list("shift", flat=True)
         return Shift.objects.filter(pk__in=shift_ids)
 
+    def get_workhour_items(self):
+        from ephios.event_management.models import AbstractParticipation
+
+        participation = (
+            self.localparticipation_set.filter(state=AbstractParticipation.States.CONFIRMED)
+            .annotate(
+                hours=ExpressionWrapper(
+                    (F("shift__end_time") - F("shift__start_time")) / 3600000000,
+                    output_field=models.DecimalField(),
+                ),
+                datetime=F("shift__start_time"),
+                reason=F("shift__event__title"),
+            )
+            .values("hours", "datetime", "reason")
+        )
+        workinghours = self.workinghours_set.all().values("hours", "datetime", "reason")
+        hour_sum = (
+            participation.aggregate(Sum("hours"))["hours__sum"]
+            + workinghours.aggregate(Sum("hours"))["hours__sum"]
+        )
+        return hour_sum, list(
+            sorted(chain(participation, workinghours), key=lambda k: k["datetime"])
```

Comment:
@felixrindt can you have a look here? I tried to union the two (identical) querysets, but the result was always empty. Is this solution worse performance-wise?